scripts/sshzobelisk

```diff
@@ -0,0 +1,40 @@
+set -euo pipefail
+
+usage()
+{
+    echo "USAGE: $0 <USERNAME ON ZOBELISK> <USER ID ON ZOBELSIK>"
+    exit 1
+}
+
+if [ $# -ne 2 ]; then
+    usage
+fi
+
+LOCAL_USER=$USER
+SERVER_USERNAME=$1
+SERVER_UID=$2
+LOCAL_REPO=$(realpath $(dirname $BASH_SOURCE)/../)
+ZOBELISK_IP=192.168.37.176
+IP=`hostname -I`
+IP=`echo $IP | head -n1 | awk '{print $1;}'` # `echo $IP | sed 's/ *$//g'`
+echo $IP
+
+sudo systemctl start ssh
+sleep 1
+
+ROS_PORT=${SERVER_UID}0
```

Comment:
Seems annoying to have user input SERVER_UID when you can just figure it out in the script

Reply:
so the script could find that out, but it would require sshing in finding the UID and then returning, settting the ros env, then sshing back in